x/protorev/keeper/epoch_hook_test.go

```diff
@@ -162,7 +162,7 @@ func (s *KeeperTestSuite) TestUpdateHighestLiquidityPools() {
 			},
 			expectedBaseDenomPools: map[string]map[string]keeper.LiquidityPoolStruct{
 				"epochTwo": {
-					"uosmo": {Liquidity: sdk.Int(sdk.NewUintFromString("999999999000000001000000000000000000")), PoolId: 49},
+					"uosmo": {Liquidity: sdk.Int(sdk.NewUintFromString("999999000000000001000000000000000000")), PoolId: 49},
```

Comment:
Note to reviewer: this number seemed right based on the migration calculations (above) and superfluid calculations (below), but admittedly I did slightly eyeball this as the underlying test components are quite obscure

(there's 0 documentation on where this number comes from, although I can infer it's the asset1 amount for a set of full range positions)